modules/ROOT/pages/authentication-authorization/dbms-administration.adoc

```diff
@@ -74,7 +74,7 @@ image::privileges_hierarchy_dbms.svg[title="DBMS privileges hierarchy"]
 The xref:authentication-authorization/built-in-roles.adoc#access-control-built-in-roles-admin[`admin` role] has a number of built-in privileges.
 These include:
 
-* Create, delete, and modify databases and aliases.
+* Create, delete, and modify databases.
```

Comment:
I think aliases should not be removed from here, because the page does talk about privileges for creating, deleting and modifying aliases.